sdks/python/apache_beam/runners/portability/flink_runner.py

```diff
@@ -42,6 +42,7 @@ def _add_argparse_args(cls, parser):
                         help='Flink version to use.')
     parser.add_argument('--flink_job_server_jar',
                         help='Path or URL to a flink jobserver jar.')
+    parser.add_argument('--artifacts-dir', default=None)
```

Comment:
Should probably be `artifacts_dir` (with an underscore)

Reply:
+1

Reply:
Done